docs/installation.rst

```diff
@@ -0,0 +1,197 @@
+  file in your :bash:`$HOME` folder:
+
+  .. code-block:: bash
+
+     export PATH="$PATH:/opt/julia-1.5.3/bin"
+
+
+Gurobi
+#######
+If you plan on using `Gurobi`_ as a solver, you will need to download and install it
+first so it can be accessed by `Jump`_. Installation of Gurobi depends on both the
+operating system and the license type. Detailed instructions can be found in the
+`Gurobi Installation Guide <https://www.gurobi.com/documentation/quickstart.html>`_.
+For Unix-like platforms, this will look like:
+
+- Choose a destination directory. For shared installation, :bash:`/opt` is recommended.
+
+  .. code-block:: bash
+
+     cd /opt
+
+- Download and unzip the package in the chosen directory:
+
+  .. code-block:: bash
+
+     wget https://packages.gurobi.com/9.1/gurobi9.1.0_linux64.tar.gz
+     tar -xvfz gurobi9.1.0_linux64.tar.gz
+
+  This will create the :bash:`/opt/gurobi910/linux64` subdirectory in which the
+  complete distribution is located.
+
+- Set environments variables. For bash users edit the **.bashrc** file in your
+  :bash:`$HOME` folder:
+
+  .. code-block:: bash
+
+     export GUROBI_HOME="/opt/gurobi910/linux64"
+     export PATH="${PATH}:${GUROBI_HOME}/bin"
+     export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${GUROBI_HOME}/lib"
+
+- The Gurobi license needs to be download and installed. Download a copy of your Gurobi
+  license from the account portal, and copy it into the parent directory of
+  :bash:`$GUROBI_HOME`.
+
+  .. code-block:: bash
+
+     cd gurobi.lic /opt/gurobi910/gurobi.lic
+
+To verify that Gurobi is properly installed, run the **gurobi.sh** shell script:
+
+.. code-block:: bash
+
+   .$GUROBI_HOME/bin/gurobi.sh
+
+
+REISE.jl
+########
+The package will need to be added to each user's default Julia environment. This can be
+done by launching Julia and typing ``]`` to access the ``Pkg`` (the built-in package
+manager) REPL environment that easily allows operations such as installing, updating
+and removing packages.
+
+.. code-block:: julia
+
+                   _
+       _       _ _(_)_     |  Documentation: https://docs.julialang.org
+      (_)     | (_) (_)    |
+       _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
+      | | | | | | |/ _` |  |
+      | | |_| | | | (_| |  |  Version 1.5.3 (2020-11-09)
+     _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
+    |__/                   |
+
+    julia> ]
+
+    pkg>
+
+From here, we recommend that you create an environment and install the dependencies
+in the same state specified in the manifest (**Manifest.toml**):
+
+.. code-block:: julia
+
+   activate /PATH/TO/REISE.jl
+   instantiate
+
+Note that the Julia packages for the user's desired solvers need to be installed
+separately. For instance, if you want to use, GLPK, the GNU Linear Programming Kit
+library, you will need to run:
+
+.. code-block:: julia
+
+  Pkg.add("GLPK")
+  import Pkg
```

Comment:
I think we can drop the comma after "use".

In the code snippet, I believe the import statement should come first.

Reply:
Oops

Reply:
Done